examples/rustls-mtls/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "rustls-mtls"
+version = "0.1.0"
+edition = "2021"
+authors = ["Rick Richardson <rick.richardson@gmail.com>"]
+
+[dependencies]
+# Remove the `provider-tls-default` feature and add `provider-tls-rustls`
```

Comment:
Since this is not a how to use rustls example, lets add some context:

```suggestion
# Remove the `provider-tls-default` feature and add `provider-tls-rustls` to use the rustls tls backend.
```

Reply:
Done.